src/Code.gs

```diff
@@ -714,13 +803,77 @@ const ChatGPTApp = (function () {
     if (response.getResponseCode() == 200) {
       let pageContent = response.getContentText();
       pageContent = convertHtmlToMarkdown(pageContent);
+      Logger.log(pageContent)
       return pageContent;
     }
     else {
+      Logger.log("no response")
       return null;
     }
   }
 
+  function getImageDescription(imageUrl, fidelity) {
+
+    if (!fidelity) {
+      let fidelity = "low";
+    }
+
+    let imageMessage = [{
+      role: "user",
+      content: [
+        {
+          type: "text",
+          text: "What is the content of this image ? Focus on important element."
```

Comment:
Better than just "What’s in this image?" (the prompt example given by openAI)

Reply:
Depends on the use case I guess. 
I found out in my tests that saying "Focus on important element" would return a more "human friendly" description, whereas it's more often a list of what is present in an image without it. 
Use case : for a screenshot of a spreadsheet with an error message
- This prompt will return a description focused on the error message that's highlighted, and from there describe what's around it. 
- The initial prompt from OpenAI gave me a more "factual" description (this a spreadsheet because we can recognize the google layout, the name of the collumns are example1 and example2 + (...) + oh and btw there is an error message that says "..."
What do you think ?

Reply:
Ok, add a comment in the code to quickly explain that (just to remember that this prompt has been tested and performed better)